great-idea/index.html

```diff
@@ -13,11 +13,92 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
+ <body>
 
-  <body>
+  <div class="container">
 
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
+   
+      <div class="top">
+          <nav class = "main-nav">
+            <a href="/home/mikaela/Desktop/UI-II-Flexbox/great-idea/services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </nav> 
+          <a href="file:///home/mikaela/Desktop/UI-II-Flexbox/great-idea/index.html">
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+          </a>
+
+      </div>
+  
+    <section class="demand">
+      <div class=demand1>
+      <h1>Innovation<br/>
+        On<br/>
+        Demand
+      </h1>
+      <img src="img/header-img.png" alt="Image of a code snippet.">
+   
+       </div>
+       <div class="start">
+       Get Started
+       </div>
+    </section>
+  
+    <div class="first-section">
+  <div class ="para">
+    <h3>Features</h3>
+    <p>
+    Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+  </div>
+  
+  <div class="para">
+    <h3>About</h3>
+    <p>
+    Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+  </div>
+  </div>
+  <div class="middle-image">
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+  </div>
+  
+<div class="second">
+  <div class="para2">
+    <h3>Services</h3>
+    <p>
+    Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+  </div>
+  
+  <div class="para2">
+    <h3>Product</h3>
+    <p>
+    Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p> 
+  </div> 
+  
+  <div class="para2">
+    <h3>Vision</h3>
+    <p>
+      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </p>
+  </div>
+  </div>
+  <div class="contact">
+    <h4>Contact</h4>
+    123 Way 456 Street <br/>
+    Somewhere, USA<br/><br/>
+    1 (888) 888-8888<br/>
+    sales@greatidea.io
+  </div>
+  <div class="copyright">
+    Copyright Great Idea! 2018
+    </div>
```

Comment:
both div's with the class of "contact" & "copyright" should be nested inside a footer. We always want a footer tag at the end of our page. Usually the footer element will include the contact info, copyright, & maybe sometimes even a nav